README.md

```diff
@@ -2,6 +2,9 @@
 
 Scans AWS EBS volumes and adds them as PersistentVolume to Kubernetes.
 
+## Warning: *Deprecated!*
+Instead we are now using Storage Classes (https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storageclasses).
```

Comment:
Hmm .. that's not the same. We don't use this anymore, because we let Kubernetes create the volume.